common/locales/en/generic.json

```diff
@@ -77,8 +77,8 @@
     "gems": "Gems",
     "gemButton": "You have <%= number %> Gems.",
     "moreInfo": "More Info",
-    "moreInfoChallengesLink": "http://habitica.wikia.com/wiki/Challenges",
-    "moreInfoTagsLink": "http://habitica.wikia.com/wiki/Tags",
+    "moreInfoChallengeURL": "http://habitica.wikia.com/wiki/Challenges",
+    "moreInfoTagURL": "http://habitica.wikia.com/wiki/Tags",
```

Comment:
You reference `moreInfoChallengesURL` and `moreInfoTagsURL` (note the extra "s"s) below.